Dockerfile

```diff
@@ -2,6 +2,13 @@ FROM python:3-alpine
 
 LABEL maintainer="CloudOps <cloudops@clark.de>"
 
+# Install kubectl
+RUN apk add --no-cache openssl curl tar gzip bash ca-certificates
+RUN curl -L -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/linux/amd64/kubectl && \
```

Comment:
Should we have the version in a var

Reply:
1.18.0 is latest though

Reply:
done :+1:

Reply:
good idea, 1.8.0 looks old, was it supposed to be 1.18 ?